articles/policies/global-limit.md

```diff
@@ -0,0 +1,241 @@
+`POST /co/verify` | Text: "Rate limit exceed"
+`GET /continue` | [Error Page](#error-page)
+`POST /continue` | [Error Page](#error-page)
+`GET /custom-login/preview` | [JSON Error](#json-error) (`too_many_requests`)
+`POST /dbconnections/delete` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /dbconnections/login` | [JSON Error](#json-error) (`too_many_requests`)
+`POST /dbconnections/login` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /dbconnections/signup` | [JSON Error](#json-error) (`too_many_requests`)
+`POST /dbconnections/signup` | [JSON Error](#json-error) (`too_many_requests`)
+`POST /dbconnections/verify_email` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /FederationMetadata/2007-06/FederationMetadata.xml` | XML Error (`fed:BadRequest`)
+`GET /i/login` | [Error Page](#error-page)
+`GET /i/login/sso/:provider` | Text: "Rate limit exceed"
+`GET /i/oauth2/authorize` |  [JSON Error](#json-error) (`access_denied`)
+`GET /login` | [Error Page](#error-page)
+`GET /login/callback` | [Error Page](#error-page)
+`POST /login/callback` | [Error Page](#error-page)
+`GET /logout` | [Error Page](#error-page)
+`POST /logout` | [Error Page](#error-page)
+`GET /mf` | [Error Page](#error-page)
+`POST /mf` | [Error Page](#error-page)
+`POST /oauth/reverse` |  [JSON Error](#json-error) (`access_denied`)
+`POST /oauth/revoke` |  [JSON Error](#json-error) (`access_denied`)
+`POST /state/introspect` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /unblock` | [Error Page](#error-page)
+`POST /unlink` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /user/ssodata` | Text: "Rate limit exceed"
+`GET /users/:id/impersonate` | [JSON Error](#json-error) (`too_many_requests`)
+`POST /users/:id/impersonate` | [JSON Error](#json-error) (`too_many_requests`)
+`GET /v2/logout` | [Error Page](#error-page)
+
+## Exceeding the Rate Limit
+If you exceed the provided rate limit for a given API endpoint, you will receive a response with [HTTP Status Code 429 (Too Many Requests)](http://tools.ietf.org/html/rfc6585#section-4) except for the cases documented above. The response will also contain the [HTTP Response Headers](policies/rate-limits#http-response-headers) which you can refer to for more information on the rate limits applicable to that endpoint.
+
+### Response body
+The response body depends on the endpoint and, in particular, the response the endpoint was meant to render originally. For example, if the response was meant to render JSON, then a JSON error response will be sent.
+
+The particular response per endpoint is stated in [Affected endpoints](#affected-endpoints) section, below you will find a description for each case.
+
+#### Error Page
+This response is sent for endpoints that render HTML content to the end user. When the rate limit is exceeded the [Error Page](https://auth0.com/docs/universal-login/custom-error-pages) will be rendered instead of the usual content.
```

Comment:
> The Error Page response is sent for endpoints that render HTML content to the end user. When you exceed the rate limit, Auth0 renders the [Error Page](https://auth0.com/docs/universal-login/custom-error-pages) instead of the expected content.